modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -53,13 +51,13 @@ The following examples cover the basic functionality of the Redpanda Schema Regi
 
 To run the sample commands and code in each example, follow these steps to set up Redpanda and other tools:
 
-. You need a running Redpanda cluster. If you don't have one, see the xref:get-started:quick-start.adoc[].
+. You need a running Redpanda cluster. If you don't have one, you can follow the xref:ROOT:get-started:quick-start.adoc[quickstart] to deploy a self-hosted cluster.
```

Comment:
We want some conditional content here. For Redpanda Cloud users, we want to link to how to create a cloud cluster.

Reply:
No need for `ROOT` because the link will only be used in the SH context.

Reply:
@JakeSCahill and @micheleRP. I've added a conditional block for Cloud content which directs users to Serverless to create a cluster if they don't have one. Is this right?